e2e/test/scenarios/visualizations-tabular/reproductions/6010-metric-filter-drill.cy.spec.js

```diff
@@ -16,30 +15,29 @@ describe("issue 6010", () => {
     cy.intercept("POST", "/api/dataset").as("dataset");
   });
 
-  // FIXME metrics v2
-  it.skip("should apply the filter from a metric when drilling through (metabase#6010)", () => {
+  it("should apply the filter from a metric when drilling through (metabase#6010)", () => {
     createMetric()
       .then(({ body: { id } }) => createQuestion(id))
       .then(({ body: { id } }) => visitQuestion(id));
 
     cartesianChartCircle().eq(0).click();
 
-    popover().findByText("See these Orders").click();
+    popover().findByText("See these Metrics").click();
     cy.wait("@dataset");
 
     cy.findByTestId("qb-filters-panel").within(() => {
-      cy.findByText("Total is greater than 150").should("be.visible");
       cy.findByText("Created At is Jan 1–31, 2024").should("be.visible");
     });
+    // FIXME metrics v2 -- check that the values in column Total are above 150
```

Comment:
I'm not sure how to do this best. Ideally, we would like to check that the filter is still in effect, but I don't think it should be listed as it's part of the metric which is still the source.